lib/migrations/add_initial_policy_machine_tables.rb

```diff
@@ -31,11 +31,5 @@ def change
     add_index :assignments, [:parent_id, :child_id], unique: true
     add_index :assignments, [:child_id]
 
-    create_table :operations_policy_element_associations, id: false do |t|
```

Comment:
Like adding a generator to drop the operations policy elements association table? I'm fine just doing that in a Dalton migration if you are.